UtopiaCity/Data/ApplicationDbContext.cs

```diff
@@ -21,11 +21,7 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         public DbSet<WeatherReport> WeatherReports { get; set; }
         public DbSet<ForPassenger> ForPassengers { get; set; }
         public DbSet<TransportManager> TransportManagers { get; set; }
```

Comment:
Not your code?

Reply:
No it doesn't refer to me.

Reply:
Well, when i merged, i got dublicate dbsets. I had to delete them.